contrib/automation_tests/core/orbit_e2e.py

```diff
@@ -0,0 +1,93 @@
+"""
+Copyright (c) 2020 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+import logging
+from typing import Iterable, Type
+import time
+
+from absl import flags
+from pywinauto import Application
+
+import orbit_testing as ot
+from core.e2e_utils import first, OrbitE2EError
+
+flags.DEFINE_boolean('dev_mode', False, 'Dev mode - expect Orbit MainWindow to be opened, '
+                                        'and do not close Orbit at the end')
+
+
+class Fragment:
+    def __init__(self, test_case: "E2ETest"):
+        self._test_case = test_case
+
+    test_case = property(lambda self: self._test_case)
+    application = property(lambda self: self._test_case.application)
+
+    def window(self, force_update=False):
```

Comment:
Does this need to be here or can it be moved to E2ETest?

Reply:
Removed